clock_exercise/StartButton.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.0
+
+Rectangle {
+    id: button
+
+    property int timeOfDissolve: 500
+    property bool enabled: true
+
+    property alias text: buttonText.text
+
+    signal buttonClicked
+
+    width: 434
+    height: 64
+    color: button.enabled? mouseArea.containsMouse? "#9900A6E2": "#00A6E2": "#0000A6E2"
+    border.color : button.enabled? mouseArea.containsMouse? "9900A6E2": "#00A6E2": "#8000A6E2"
+    border.width : 1
+    radius: 32
+
+    Behavior on color {
+        ColorAnimation {
+            duration: timeOfDissolve
+        }
+    }
+
+    Text {
+        id: buttonText
+
+        width: 98
+        height: 36
+        color: button.enabled? mouseArea.containsMouse? "#99FFFFFF": "#FFFFFF": "#8000A6E2"
+        anchors.centerIn: parent
+        font.pixelSize: 30
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Buenos Aires"
+        font.styleName: "SemiBold"
+
+        Behavior on color {
+            ColorAnimation {
+                duration: timeOfDissolve
+            }
+        }
+    }
+
+    MouseArea {
+        id: mouseArea
+
+        hoverEnabled: true
+        anchors.fill: parent
+        onClicked: if(button.enabled) button.buttonClicked()
```

Comment:
Potrei ricordarmi male, ma la property `enabled`, così come `visible`, agiscono automaticamente su tutto l'albero del componente, quindi il controllo su `button.enabled` qua non è necessario perché quando è disabilitato l'elemento root è disabilitata anche questa MouseArea

Reply:
Testato, il controllo ci vuole

Reply:
Il problema in questo caso è diverso: hai creato una nuova property `enabled` che va a sovrascrivere quella presente di default!
Se rimuovi la property che hai aggiunto tu, e setti per prova `enabled: false` rimuovendo il check, vedrai che il segnale non viene emesso.